presto-main/src/main/java/com/facebook/presto/execution/buffer/ClientBuffer.java

```diff
@@ -371,7 +376,7 @@ private synchronized BufferResult processRead(long sequenceId, DataSize maxSize)
             }
             result.add(page.getSerializedPage());
         }
-        return new BufferResult(taskInstanceId, sequenceId, sequenceId + result.size(), false, result);
+        return new BufferResult(taskInstanceId, sequenceId, sequenceId + result.size(), false, bufferedBytes, result);
```

Comment:
Does the client expect `bufferedBytes` to include the size of the response pages? Or should it contain the remaining buffered bytes (excluding the size of pages returned)?

Reply:
@tdcmeehan I think the question from @arhimondr is whether the remaining bytes header should include the current page being returned.  The answer is no.  Can you double check we do not include the size of current page being returned in the response?

Reply:
In the original design and Prestissimo implementation, the remaining bytes do not include the current response pages.  It is normal (even expected) that the remaining bytes are different between the HEAD and GET calls.

Reply:
Updated to remove the bytes from results pages.